contracts/contracts/utilities/Params.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import { IMACI } from "../interfaces/IMACI.sol";
+import { AccQueue } from "../trees/AccQueue.sol";
+import { TopupCredit } from "../TopupCredit.sol";
+import { VkRegistry } from "../VkRegistry.sol";
+
+/// @title Params
+/// @author PSE
+/// @notice This contracts contains a number of structures
+/// which are to be passed as parameters to Poll contracts.
+/// This way we can reduce the number of parameters
+/// and avoid a stack too deep error during compilation.
```

Comment:
I don't understand this sentence.

Reply:
Basically in the EVM there's a limit to how many local variables you can have (params and return values included) - so basically instead of passing 4 params for tree depths (as an example), we pass only one (the struct) - hope this makes sense

Reply:
It's clearer when you explain it :)
then maybe it should be **avoid a stack of deep errors during compilation**?

Reply:
"stack too deep" is the actual error name, you can read more [here](https://soliditydeveloper.com/stacktoodeep) btw